pkgs/applications/finance/irpf/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib, stdenvNoCC, jdk11, makeWrapper, hicolor-icon-theme, fetchzip, fetchgit }:
+
+let
+  pkgbuild = fetchgit rec {
+    url = "https://aur.archlinux.org/irpf.git/";
+    rev = "dc76a6366de72e236c91986a4c2a467d2472749f";
+    sha256 = "1hql5ljwq8g9h0kbgpfyss7mcmp58pzvjs5dydxcs39lxcnf972c";
+  };
+in
+stdenvNoCC.mkDerivation rec {
+  pname = "irpf";
+  version = "2022.1.0";
+
+  src = let
+    year = lib.head (lib.splitVersion version);
+  in fetchzip {
+    url = "https://downloadirpf.receita.fazenda.gov.br/irpf/${year}/irpf/arquivos/IRPF${year}-1.0.zip";
+    sha256 = "0h8f51ilvg7m6hlx0y5mpxhac90p32ksbrffw0hxdqbilgjz1s68";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+  propagatedBuildInputs = [ hicolor-icon-theme ];
+
+  installPhase = ''
+    mkdir -p "$out/share/icons/hicolor/128x128/apps"
+    mkdir -p "$out/share/applications"
+    mkdir -p "$out/share/licenses/${pname}"
+    mkdir -p "$out/share/${pname}"
+    mkdir "$out/bin"
+    BASEDIR="$out/share/${pname}"
+
+    cp -r help lib lib-modulos "$BASEDIR"
+
+    install -Dm755 irpf.jar "$BASEDIR/${pname}.jar"
+    install -Dm644 Leia-me.htm offline.png online.png pgd-updater.jar "$BASEDIR"
+
+    makeWrapper ${jdk11}/bin/java $out/bin/${pname} \
+      --add-flags "-Dawt.useSystemAAFontSettings=on \
+      -Dswing.aatext=true \
+      -jar $BASEDIR/${pname}.jar" \
+      --prefix _JAVA_AWT_WM_NONREPARENTING=1 AWT_TOOLKIT=MToolkit
+
+    install -Dm644 ${pkgbuild}/irpf.png $out/share/icons/hicolor/128x128/apps/
+    install -Dm644 ${pkgbuild}/irpf.desktop $out/share/applications/
+    install -Dm644 ${pkgbuild}/Copyright $out/share/licenses/${pname}/
```

Comment:
```suggestion
    install -Dtm644 $out/share/icons/hicolor/128x128/apps ${pkgbuild}/irpf.png
    install -Dtm644 $out/share/applications ${pkgbuild}/irpf.desktop
    install -Dtm644 $out/share/licenses/${pname} ${pkgbuild}/Copyright
```

Reply:
I tried this and it didn't work, but I implemented something that will.